x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/GetStackTracesRequest.java

```diff
@@ -32,7 +35,7 @@
 /**
  * A request to get profiling details
  */
-public class GetStackTracesRequest extends ActionRequest implements IndicesRequest {
+public class GetStackTracesRequest extends ActionRequest implements IndicesRequest.Replaceable {
```

Comment:
Implementing `IndicesRequest.Replaceable` indicates to infrastructure in Elasticsearch that we want to support wildcards, see also https://github.com/elastic/elasticsearch/blob/510c8515abecf5ca23154e25a72247329e8aa311/x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/IndicesAndAliasesResolver.java#L136-L146